golang/cosmos/app/app.go

```diff
@@ -231,8 +243,15 @@ func init() {
 
 // NewGaiaApp returns a reference to an initialized Gaia.
 func NewGaiaApp(
-	logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bool, skipUpgradeHeights map[int64]bool,
-	homePath string, invCheckPeriod uint, encodingConfig gaiaappparams.EncodingConfig, appOpts servertypes.AppOptions, baseAppOptions ...func(*baseapp.BaseApp),
+	logger log.Logger,
+	db dbm.DB, traceStore io.Writer,
```

Comment:
_Sic._

I'm replying _sic._ since it's the way Gaia is formatted, and I don't want to introduce any formatting changes that aren't semantic, or else the next 3-way merge will have unnecessary conflicts.

Reply:
Totally fine. Thanks for the explanation.

Reply:
And kudos for correct use of the Latin idiom!!